libs/stylelint-config/index.js

```diff
@@ -5,13 +5,95 @@ module.exports = {
   extends: "stylelint-config-property-sort-order-smacss",
   plugins: [
     "stylelint-scss",
-    "stylelint-use-nesting",
-    //"stylelint-selector-no-empty",  => uncomment this line once this library is again compatible with stylelint v15+
+    "@stylistic/stylelint-plugin",
+    // "stylelint-use-nesting",     => uncomment this line once this library is again compatible with stylelint v16+
+    //"stylelint-selector-no-empty",  => uncomment this line once this library is again compatible with stylelint v16+
     "stylelint-group-selectors",
     "stylelint-max-lines",
     "stylelint-declaration-block-no-ignored-properties"
   ],
   rules: {
```

Comment:
Is it worth considering the implementation of a new stylelint rule lightness-notation? https://stylelint.io/user-guide/rules/lightness-notation/ ?